test/setup.js

```diff
@@ -13,14 +13,14 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Why is this necessary?

Reply:
This is necessary because of the closest `package.json` in the test folder.
but maybe we can just add the dependencies on the `package.json` it self =(